current_version.json

```
@@ -1,13 +1,13 @@
 {
-    "tag_name": "v1.0.2",
+    "tag_name": "v1.0.7",
     "assets": [
-        {
-            "browser_download_url": "https://github.com/FriendsOfGalaxy/galaxy-integration-steam/releases/download/v1.0.2/windows.zip",
-            "name": "windows.zip"
-        },
-        {
-            "browser_download_url": "https://github.com/FriendsOfGalaxy/galaxy-integration-steam/releases/download/v1.0.2/macos.zip",
-            "name": "macos.zip"
-        }
+		{
+			"browser_download_url": "https://github.com/ABaumher/galaxy-integration-steam/releases/download/v1.0.7/windows.zip",
+			"name": "windows.zip"
+		},
+		{
+			"browser_download_url": "https://github.com/ABaumher/galaxy-integration-steam/releases/download/v1.0.7/mac.zip",
+			"name": "mac.zip"
+		}
```

```suggestion
		{
			"browser_download_url": "https://github.com/GOG-Nebula/galaxy-integration-steam/releases/download/v1.0.7/windows.zip",
			"name": "windows.zip"
		},
		{
			"browser_download_url": "https://github.com/GOG-Nebula/galaxy-integration-steam/releases/download/v1.0.7/mac.zip",
			"name": "mac.zip"
		}
```

dw about squashing this or anything just make it another commit if you want to

---

Once they're added to releases here that'll need to be updated. Leaving it for now so i don't have to get another review, lol